src/Extractor/HttpClient.php

```diff
@@ -65,6 +65,9 @@ public function __construct(Client $client, $config = [], LoggerInterface $logge
             'timeout' => 10,
             // number of redirection allowed until we assume request won't be complete
             'max_redirect' => 10,
+			
+			// Additional Guzzle configs.
+			'config' => []
```

Comment:
This additional config is too vague to be merged.
Maybe we can define a more appropriate config, like, I don't know, `proxy_settings`, etc.